code/datums/components/material_container.dm

```diff
@@ -19,6 +19,7 @@
 	var/list/allowed_typecache
 	var/last_inserted_id
 	var/precise_insertion = FALSE
+	var/bump_loading = TRUE
```

Comment:
This probably shouldn't be true by default.

Reply:
Why not?

Reply:
Wouldn't this allow literally any machine that uses this to be bumploaded by default, Ore silos, Protolathes, destructive analyzers etc.

Reply:
Yes?

Reply:
So remove it then.

Reply:
Also there's an issue with some freezers placed on top on walls; specifically, items don't bump into them at all. Not sure how to fix this.

Reply:
Map design issue.

Reply:
Should I adress it in this PR?

Reply:
IIRC this map fix was made so that you could not see through freezers the other side of the wall.

Reply:
You could add a bump check for closed turfs that checks for machines on the same tile

Reply:
I solved this by making freezers opaque and removing the overlapping walls.